salesforce-native/src/main/java/io/ballerinax/salesforce/ReadOperationExecutor.java

```diff
@@ -0,0 +1,121 @@
+package io.ballerinax.salesforce;
```

Comment:
Add missing license

Reply:
Added in 8bfb122ccf270a0868c40a7ea6ddaef1d3a79a39